services/shotgrid_common/ayon_shotgrid_hub/update_from_shotgrid.py

```diff
@@ -284,7 +286,7 @@ def update_ayon_entity_from_sg_event(
     )
 
     if not ay_entity:
-        raise ValueError("Unable to update a non existing entity.")
+        raise MissingParentError("Unable to update a non existing entity.")
```

Comment:
This may clarify where the entity is missing perhaps?
```suggestion
        raise MissingParentError("Unable to update a non existing AYON entity.")
```